prime-dotnet-webapi/ViewModels/Enrollee/Constants.cs

```diff
@@ -0,0 +1,7 @@
+namespace Prime.ViewModels
+{
+    public static class Constants
+    {
+        public const string PaperGpidPrefix = "NOBCSC";
+    }
+}
```

Comment:
I don't think creating a new global `Constants` class is the best way to do this; and the ViewModels folder is not the right place to put it.

I could see it being added to `PrimeConfiguration`, but really it's an implementation detail of how we are currently distinguishing regular `Enrollee`s from paper `Enrollee`s. Previously I spent some time thinking about how we could have the `enrollee.GPID.StartsWith("NOBCSC")` predicate on the `PaperEnrolleeService` for use in other services but ultimately decided it wasn't worth it. 

I feel either the `PaperEnrolleeService` or the `Enrollee` model should own this constant. I first thought that the service was the correct place for it but now I'm leaning more toward putting it on the `Enrollee` model. At some point in the future an `enrollee.IsPaper` property could be added or something similar.

Reply:
Moving the constant to `Enrollee` model for now since it's shared by both `EnrolleeService` and `ErnolleePaperSumissionService`